docs/rules/id-match.md

```diff
@@ -29,6 +29,11 @@ This rule needs a text RegExp to operate with, and accepts an options map. Its s
 1. `true` is the default and checks all property names
 2. `false` does not check property names at all (default)
 
+`onlyDeclaration` can have the following values:
+
```

Comment:
If this is a request to action something, then please phrase it that way. Now I cannot tell if you are thinking out loud, waiting for someone else to approve what you have said or actioning it.